### ISSUE TITLE
AnimDessin2 Not Installing in Photoshop 2021

### ISSUE THREAD

**Issue by plindale** [sbaril/Photoshop-Animation#12]
I had it installed fine in v.2020 on my MacBook Pro (Catalina) but when Adobe updated to 2021 it refused to appear in the Extensions (legacy) menu item. I've updated Anastaisy's Extension Manager to the latest version and it goes through the motions of installing, says it was successful, but then nothing? Same for AnimCouleur2.

Any suggestions guys?

**Comment by plindale** [comment]
Found the fix: In Anastasiy's Extension Manager, choose file menu -> reset extension list. After you do this the extension will install just fine. Though I did have to reinstall twice to get it to work. The first time around when I created a new animation document the new layer was not placed in an Animalities group and neither was any new layer after that unless I specifically created a new layer group. Also, the option to specify a frame rate did not appear. Upon installing a second time both of these issues appeared to resolved themselves. Maybe something to look at Stephane.
